internal/app/observer/collecting/deposit.go

```diff
@@ -171,6 +172,15 @@ func (c *DepositCollector) processGenesisRecord(ctx context.Context, rec *observ
 				hrd, _ = pulse.Number(pulse.MinTimePulse).AsApproximateTime()
 			}
 
+			var vestingType models.DepositType
+			switch depositState.VestingType {
+			case 0:
```

Comment:
0 -	DefaultVesting VestingType = iota
	// Deprecated: Never used
1 -	Vesting1
	// Vesting type for funds (usually with zero vesting period and zero-step)
2 -	Vesting2
	// Deprecated: Never used
3 -	Vesting3
	// Deprecated: Never used
4 -	Vesting4
5 - ...